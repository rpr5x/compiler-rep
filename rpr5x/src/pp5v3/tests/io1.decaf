string s;

void main()
{
   int n;

    ReadLine();
    s = ReadLine();
    Print(s, "\n");
    Print(ReadLine(), "\n");
    ReadInteger();
    n = ReadInteger();
    Print(n, "\n");
    Print(ReadInteger()*2, "\n");
}