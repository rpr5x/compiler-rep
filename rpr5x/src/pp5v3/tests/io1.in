First
CS143 rocks!
hello world
45
-234
50

