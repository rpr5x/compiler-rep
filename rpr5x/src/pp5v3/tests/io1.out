Loaded: /if14/ww6r/WorkingDisk/TA/compiler/exceptions/exceptions.s
CS143 rocks!
hello world
-234
100
